worker/internal/app/app.go

```diff
@@ -1,15 +1,87 @@
 package app
 
 import (
-	"fmt"
+	"context"
+	"errors"
+	"log"
 	"net/http"
+
+	"github.com/labstack/echo/v4"
+	"github.com/labstack/echo/v4/middleware"
+	rlog "github.com/reearth/reearthx/log"
+	"github.com/reearth/reearthx/rerror"
+	"go.opentelemetry.io/contrib/instrumentation/github.com/labstack/echo/otelecho"
 )
 
-func handler(w http.ResponseWriter, r *http.Request) {
-	fmt.Fprintf(w, "Hello, World")
+func initEcho(ctx context.Context, cfg *ServerConfig, handler *Handler) *echo.Echo {
+	if cfg.Config == nil {
+		log.Fatalln("ServerConfig.Config is nil")
+	}
+
+	e := echo.New()
+	e.Debug = cfg.Debug
+	e.HideBanner = true
+	e.HidePort = true
+	e.HTTPErrorHandler = errorHandler(e.DefaultHTTPErrorHandler)
+
+	// basic middleware
+	logger := rlog.NewEcho()
+	e.Logger = logger
+	e.Use(middleware.Recover(), otelecho.Middleware("reearth-cms"))
+
+	api := e.Group("/api")
+
+	api.GET("/ping", func(c echo.Context) error { return c.JSON(http.StatusOK, "pong") })
+
+	t := handler.DecompressHandler()
+	api.GET("/decompress", t)
```

Comment:
```suggestion
```

GET should not cause side effects.

Reply:
Opps, sorry I forgot to change it to POST since it's easier to Debug with using GET. I'll fix it!